receiver/kubernetesclusterreceiver/collector.go

```diff
@@ -0,0 +1,213 @@
+// Copyright 2020 OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kubernetesclusterreceiver
```

Comment:
Since the current processor is called k8sprocessor (Go pkg name) and k8s_tagger (processor name) let's keep with the convention and call the package k8sclusterreceiver and k8s_cluster.

Reply:
Renamed package to `k8sclusterreceiver `